src/library/commissioner_safe.cpp

```diff
@@ -89,7 +95,7 @@ void CommissionerSafe::StartEventLoopThread()
 
     mEventThread = std::thread([this]() {
         LOG_INFO(LOG_REGION_MESHCOP, "event loop started in background thread");
-        event_base_loop(mEventBase.Get(), EVLOOP_NO_EXIT_ON_EMPTY);
```

Comment:
EVLOOP_NO_EXIT_ON_EMPTY not working?

Reply:
It can work for now. But it is a relatively new feature that some platform with an older `libevent` don't have support of this. We may downgrade the `libevent` library version later to support those platforms.